civis/utils/_jobs.py

```diff
@@ -26,7 +30,41 @@ def run_job(job_id, api_key=None, client=None):
     if client is None:
         client = APIClient(api_key=api_key, resources='all')
     run = client.jobs.post_runs(job_id)
-    return CivisFuture(client.jobs.get_runs,
-                       (job_id, run['id']),
-                       client=client,
-                       poll_on_creation=False)
+    fut = CivisFuture(client.jobs.get_runs,
+                      (job_id, run['id']),
+                      client=client,
+                      poll_on_creation=False)
+    if wait:
+        fut.result()
+    return fut
```

Comment:
The `result` method is blocking. Maybe this is unnecessary and `run_job(job_id).result()` is good enough. ... In fact, that's about the same number of extra characters as `run_job(job_id, wait=True)`. Okay, I can drop the `wait` argument.